### ISSUE TITLE
Are there overlay maps?

### ISSUE THREAD

**Issue by Siecje** [graememcc/micropolisJS#32]
Can you see police coverage, Pollution, etc?

**Comment by graememcc** [comment]
No, that's one thing I never got around to when porting Micropolis to JS. It remains on my todo list, although I'm afraid I'm not sure when I'll have time to get to it. Sorry I don't have better news.
